src/components/tasks/card/index.tsx

```diff
@@ -31,6 +31,9 @@ import { useEditMode } from '@/hooks/useEditMode';
 import { useGetUsersByUsernameQuery } from '@/app/services/usersApi';
 import { ConditionalLinkWrapper } from './ConditionalLinkWrapper';
 import { isNewCardDesignEnabled } from '@/constants/FeatureFlags';
+import SuggestionBox from '../SuggestionBox/SuggestionBox';
+import { userDataType } from '@/interfaces/user.type';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
 
 type Props = {
     content: task;
```

Comment:
please import types from .type file and use it here

Reply:
This is the way, it has been done across the whole repo. So i went along with it @Pratiyushkumar